tensorboard/webapp/metrics/views/metrics_container.ts

```diff
@@ -17,10 +17,12 @@ import {ChangeDetectionStrategy, Component} from '@angular/core';
 @Component({
   selector: 'metrics-dashboard',
   template: `
-    <nav>
-      <runs-selector></runs-selector>
-    </nav>
-    <metrics-main-view></metrics-main-view>
+    <div class="plugin-view">
```

Comment:
I'm not seeing why this extra wrapper was necessary.  Can we drop it, or if not, could you add a comment explaining why it's needed?

Reply:
Removed. It was needed at one point but now can be removed.